Installers/Windows/MonoGame.nsi

```diff
@@ -272,6 +288,7 @@ LangString VS2012Desc ${LANG_ENGLISH} "Install the project templates for Visual
 LangString VS2013Desc ${LANG_ENGLISH} "Install the project templates for Visual Studio 2013"
 LangString VS2015Desc ${LANG_ENGLISH} "Install the project templates for Visual Studio 2015"
 LangString VS2017Desc ${LANG_ENGLISH} "Install the project templates for Visual Studio 2017"
+LangString VS2017Desc ${LANG_ENGLISH} "Install the project templates for Visual Studio 2019"
```

Comment:
Shouldn't it be `VS2019Desc` instead of `VS2017Desc`?

Reply:
Done